builtin_interfaces/Time stamp
float32 value
string sensor_name
uint32 incubator_id
